docs/ug/configFiles.md

```diff
@@ -0,0 +1,183 @@
+
+<br>
+
+Column Name | Explanation
+----------- | -----------
+Repository's Location {{ mandatory }}| The `GitHub URL` or `Disk Path` to the git repository e.g., `https://github.com/foo/bar.git` or `C:\Users\user\Desktop\GitHub\foo\bar`
+Branch | The branch to analyze in the target repository e.g., `master`. Default: the default branch of the repo
+File formats<sup>*+</sup> | The file extensions to analyze. Default: all file formats
+Ignore Glob List<sup>*+</sup> | The list of file path globs to ignore during analysis for each author. e.g., `test/**;temp/**`
+Ignore standalone config | To ignore the standalone config file (if any) in target repository, enter **`yes`**. If the cell is empty, the standalone config file in the repo (if any) will take precedence over configurations provided in the csv files.
+Ignore Commit List<sup>*+</sup> | The list of commits to ignore during analysis. For accurate results, the commits should be provided with their full hash.
+Ignore Authors List<sup>*+</sup> | The list of authors to ignore during analysis. Authors should be specified by their [Git Author Name](#a-note-about-git-author-name).
+
+<sup>* **Multi-value column**: multiple values can be entered in this column using a semicolon `;` as the separator.</sup>
+<sup>+ **Overrideable column**: prepend with `override:` to use entered value(s) instead of value(s) from standalone config.</sup>
+
+<!-- ==================================================================================================== -->
+
+## `author-config.csv`
+
+Optionally, you can use a `author-config.csv` (which should be in the same directory as `repo-config.csv` file) to provide more details about the authors to analyze ([example](author-config.csv)). It should contain the following columns:
+
+Column Name | Explanation
+----------- | -----------
+Repository's Location | Same as `repo-config.csv`. Default: all the repos in `repo-config.csv`
+Branch | The branch to analyze for this author e.g., `master`. Default: the author will be bound to all the repos in `repo-config.csv` that has the same repo's location, irregardless of branch
+Author's GitHub ID {{ mandatory }}| GitHub username of the target author e.g., `JohnDoe`
+Author's Emails<sup>*</sup> | Associated Github emails of the author. This can be found in your [GitHub settings](https://github.com/settings/emails).
+Author's Display Name | The name to display for the author. Default: author's GitHub username.
+Author's Git Author Name<sup>*</sup> | The meaning of _Git Author Name_ is explained in [_A note about git author name_](#a-note-about-git-author-name).
+Ignore Glob List<sup>*</sup> | Files to ignore for this author, in addition to files ignored by the patterns specified in `repo-config.csv`
+
+<sup>* **Multi-value column**: multiple values can be entered in this column using a semicolon `;` as the separator.</sup>
+
+> Note: the first row consists of config headings, which is ignored by RepoSense.
```

Comment:
A minor comment: there is a difference in style for these two instances of the same note.

![image](https://user-images.githubusercontent.com/35621759/84052048-9c683700-a9e2-11ea-80af-7fd953c65f37.png)

We can standardise the style for the note.

Reply:
We could also remove the note entirely, as it has already been stated for `repo-config.csv`